data/Application.css

```
@@ -19,3 +19,30 @@ map-point {
         transform: scale(1);
     }
 }
+
+button.osd {
```

```suggestion

/* TODO Remove when https://github.com/elementary/granite/issues/897 is fixed */
button.osd {
```

We should explicit a TODO comment then